### DISCUSSION TITLE
Inertia SSR on Laravel 8 Jetstream

### DISCUSSION THREAD

**Discussion by glennraya** [inertiajs/inertia#1078] [Help]
Hi, I have a Laravel 8 with Jetstream/InertiaJS installed. I was trying to implement the SSR on Laravel Jetstream (Inertia Stack) and I can't seem to have it work even though I followed every step in the documentation. Does anybody here having similar issue? and if you got it working, could you please help? Thanks!

**Comment by TeslaLyon** [comment]
I may have encountered the same problem as you, did you solve the problem?

**Reply by glennraya** [reply]
Hi, Laravel 9 with an updated Jetstream now natively supports SSR. But I haven't figured it out on Laravel 8.
